corehq/apps/export/tests/test_export_conversion.py

```diff
@@ -133,6 +145,22 @@ def test_nested_repeat_conversion(self):
         self.assertEqual(column.label, 'Modified Nested')
         self.assertEqual(column.selected, True)
 
+    def test_transform_conversion(self):
```

Comment:
this one is tough because of our current `get_column` implementation:
```
def get_column(self, item_path, item_transform):
        # Columns should be unique by item path and item transform
        for column in self.columns:
            if column.item.path == item_path:
                transform = column.item.transform if isinstance(column.item, SystemExportItem) else None
                if transform == item_transform:
                    return column
        return None
```
things can have multiple transforms so it's weird to pass in just one transform (export column has a list of transforms). once system props are just columns, we can pass an array of transforms